.github/workflows/translations-sync.yml

```diff
@@ -3,27 +3,27 @@
 
 # See translations-upload.yml for automation to upload our source content
 # See translations-pr-lint.yml for quality control we conduct on ingress of new translations.
-name: Crowdin Download
+name: Crowdin Translations Sync
 
 on:
-  workflow_dispatch: # Allow running when we want to, for events such as urgent translation mistakes or 100% completed languages
+  workflow_dispatch: # Manual trigger for urgent fixes
```

Comment:
100% completed languages would be out of band, off Friday deployments. If we want to remove that self-documenting use case, I suppose that's fine.